openapi-docs/docs/DefaultApi.md

```diff
@@ -0,0 +1,132 @@
+# AmbianicServerlessFunctionsApiDefinition.DefaultApi
+
+All URIs are relative to *https://ambianic-functions.netlify.app*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**netlifyFunctionsCustomersGet**](DefaultApi.md#netlifyFunctionsCustomersGet) | **GET** /.netlify/functions/customers | 
+[**subscriberPost**](DefaultApi.md#subscriberPost) | **POST** /subscriber | 
+[**subscriptionDelete**](DefaultApi.md#subscriptionDelete) | **DELETE** /subscription | 
+
+
+## netlifyFunctionsCustomersGet
+
+> netlifyFunctionsCustomersGet()
+
+
+
+Serverless function to fetch all customers from a stripe service account.
+
+### Example
+
+```javascript
+import AmbianicServerlessFunctionsApiDefinition from 'ambianic_serverless_functions_api_definition';
+
+let apiInstance = new AmbianicServerlessFunctionsApiDefinition.DefaultApi();
+apiInstance.netlifyFunctionsCustomersGet((error, data, response) => {
+  if (error) {
+    console.error(error);
+  } else {
+    console.log('API called successfully.');
+  }
+});
+```
+
+### Parameters
+
+This endpoint does not need any parameter.
```

Comment:
still a mess

Reply:
Parameters for the three endpoints have been added.